fv3core/fv3core/stencils/delnflux.py

```diff
@@ -13,9 +13,9 @@
 )
 
 import pace.dsl.gt4py_utils as utils
-from fv3core.utils.grid import DampingCoefficients, axis_offsets
 from pace.dsl.stencil import StencilFactory, get_stencils_with_varied_bounds
 from pace.dsl.typing import FloatField, FloatFieldIJ, FloatFieldK
+from pace.stencils.testing.grid import DampingCoefficients, axis_offsets
```

Comment:
I see most stencil files import from pace.stencils.testing.grid, is there a way to move whatever they’re using out of testing while keeping our testing-only files there? GridData, axis_offsets, and DampingCoefficients don't really belong in `testing`.